compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -823,14 +847,19 @@ static const char *getAMDProcessorTypeAndSubtype(
       // Models 60h-6Fh (Krackan1).
       // Models 70h-77h (Sarlak).
       CPU = "znver5";
-      *Subtype = AMDFAM1AH_ZNVER5;
+      Subtype = AMDFAM1AH_ZNVER5;
       break; //  "znver5"
     }
     break;
   default:
     break; // Unknown AMD CPU.
   }
 
+  if (Type != CPU_TYPE_MAX)
+    __cpu_model.__cpu_type = Type;
+  if (Subtype != CPU_SUBTYPE_MAX)
+    __cpu_model.__cpu_subtype = Subtype;
```

Comment:
Likewise

Reply:
I don't follow, I think one of your comments wasn't posted

Reply:
Ugh, silly GitHub. I was thinking early return might be useful here.